main.go

```diff
@@ -25,11 +26,14 @@ func HealthCheckHandler(w http.ResponseWriter, r *http.Request) {
 	renderer.JSON(w, http.StatusOK, map[string]string{"status": "OK"})
 }
 
-func BuildHTTPMux(arbiterURL, contentStoreURL string) *http.ServeMux {
-	httpMux := http.NewServeMux()
-	httpMux.HandleFunc("/healthcheck", HealthCheckHandler)
+func BuildHTTPMux(arbiterURL, contentStoreURL string) http.Handler {
+	httpMux := mux.NewRouter()
+	httpMux.Methods("GET").Path("/healthcheck").HandlerFunc(HealthCheckHandler)
 	contentStoreHandler := NewContentStoreHandler(arbiterURL, contentStoreURL)
-	httpMux.HandleFunc("/content/", contentStoreHandler.PutContentItem)
+	httpMux.Methods("PUT").PathPrefix("/content{base_path:.*}").HandlerFunc(contentStoreHandler.PutContentStoreRequest)
+	httpMux.Methods("PUT").PathPrefix("/publish-intent{base_path:.*}").HandlerFunc(contentStoreHandler.PutContentStoreRequest)
+	httpMux.Methods("GET").PathPrefix("/publish-intent{base_path:.*}").HandlerFunc(contentStoreHandler.GetContentStoreRequest)
+	httpMux.Methods("DELETE").PathPrefix("/publish-intent{base_path:.*}").HandlerFunc(contentStoreHandler.DeleteContentStoreRequest)
```

Comment:
Now that these are capturing the `base_path`, there's no need for them to be `PathPrefix`'s.

Also it may be worth updating the regex used to require a leading `/` on the base_path.  This would match the routing in content-store etc.